Zend/zend_vm_def.h

```diff
@@ -9549,6 +9549,82 @@ ZEND_VM_HANDLER(202, ZEND_CALLABLE_CONVERT, UNUSED, UNUSED)
 	ZEND_VM_NEXT_OPCODE();
 }
 
+ZEND_VM_HANDLER(204, ZEND_FRAMELESS_ICALL_0, UNUSED, UNUSED)
+{
+	USE_OPLINE
+	SAVE_OPLINE();
+	zend_frameless_function_0 function = (zend_frameless_function_0)zend_flf_handlers[opline->extended_value];
+	function(EX_VAR(opline->result.var));
+	if (EG(exception)) {
+		ZVAL_NULL(EX_VAR(opline->result.var));
+		HANDLE_EXCEPTION();
+	}
+	ZEND_VM_NEXT_OPCODE();
+}
+
+ZEND_VM_HANDLER(205, ZEND_FRAMELESS_ICALL_1, CONST|TMP|VAR|CV, UNUSED)
+{
+	USE_OPLINE
+	SAVE_OPLINE();
+	zend_frameless_function_1 function = (zend_frameless_function_1)zend_flf_handlers[opline->extended_value];
+	zval *arg1 = GET_OP1_ZVAL_PTR_DEREF(BP_VAR_R);
+	if (EG(exception)) {
+		ZVAL_NULL(EX_VAR(opline->result.var));
+		HANDLE_EXCEPTION();
+	}
```

Comment:
Not every specialization needs an exception check after getting the op. Only CV I believe?

Reply:
Ah, right. I forgot to check this.